src/main/java/io/confluent/connect/elasticsearch/bulk/BulkProcessor.java

```diff
@@ -391,6 +405,45 @@ private BulkResponse execute() throws Exception {
         }
       }
     }
+
+    private void handleMalformedDoc(BulkResponse bulkRsp) {
+      // if the elasticsearch request failed because of a malformed document,
+      // the behavior is configurable.
+      switch (behaviorOnMalformedDoc) {
+        case WARN:
+          log.warn("Encountered an illegal document error when executing batch {} of {}"
+                  + " records. Error was {}. Will not index record.",
+              batchId, batch.size(), bulkRsp.getErrorInfo());
+          return;
+        case IGNORE:
+          if (log.isDebugEnabled()) {
+            log.debug("Encountered an illegal document error when executing batch {} of {}"
+                    + " records. Ignoring. Error was {}",
```

Comment:
Ideally it would match the format of the message above. `Error was {}. Ignoring and will not index record.`